src/components/Player/styles/_search.scss

```diff
@@ -0,0 +1,5 @@
+.search-wrapper {
+    border: solid 2px pink;
+    grid-column: 1 / 2;
+    text-align: center;
+}
```

Comment:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file